Dockerfile

```diff
@@ -76,14 +81,18 @@ COPY requirements.txt ./
 RUN pip install -r requirements.txt
 
 # COPY the local state.
-COPY Makefile schema.sql ./
+COPY Makefile ./
 COPY css/ css/
 COPY db/ db/
 COPY html/ html/
 COPY img/ img/
 COPY js/ js/
 COPY py/ py/
 
+# Fix some missing deps in ord-schema.
+# TODO(skearnes): Move into ord-schema requirements.txt.
+RUN pip install pillow rdkit-pypi
```

Comment:
Does this code change get rid of conda completely? AFAIK, the pypi version of RDKit isn't always going to be updated and maintained. I feel like we should keep conda.

Reply:
Done.